assets/js/blocks/cart-checkout/checkout/frontend.js

```diff
@@ -7,13 +7,28 @@ import { withRestApiHydration } from '@woocommerce/block-hocs';
  * Internal dependencies
  */
 import Block from './block.js';
+import blockAttributes from './attributes';
 import renderFrontend from '../../../utils/render-frontend.js';
 
 const getProps = ( el ) => {
+	const attributes = {};
+
+	Object.keys( blockAttributes ).map( ( key ) => {
+		if ( typeof el.dataset[ key ] !== 'undefined' ) {
+			if (
+				el.dataset[ key ] === 'true' ||
+				el.dataset[ key ] === 'false'
+			) {
+				attributes[ key ] = el.dataset[ key ] !== 'false';
+			} else {
+				attributes[ key ] = el.dataset[ key ];
+			}
+		}
+		return true;
```

Comment:
Because of linting.

> Since map builds a new array, using it when you aren't using the returned array is an anti-pattern; use forEach or for-of instead.

Will swap it out.